obal/data/roles/build_package/tasks/download.yml

```diff
@@ -1,5 +1,23 @@
-- name: "Download {{ build_package_koji_command }}  task(s) logs"
+- name: "Download {{ build_package_koji_command }} task(s) logs"
   command: "{{ build_package_koji_command }}  download-logs -r {{ build_package_koji_tasks | join(' ') }}"
   args:
     chdir: "{{ inventory_dir }}"
-  when: build_package_build_system == 'koji'
+  when: build_package_build_system == 'koji' and build_package_download_logs
+
+- block:
+    - name: 'Create RPM downloads directory'
+      file:
+        state: directory
+        path: "{{ inventory_dir }}/downloaded_rpms"
+
+    - name: "Download {{ build_package_koji_command }} RPM(s)"
+      command: "{{ build_package_koji_command }} download-task {{ item }}"
+      args:
+        chdir: "{{ inventory_dir }}/downloaded_rpms"
+      with_items: "{{ build_package_koji_tasks }}"
+
+    - name: 'Create repo from downloaded RPMs'
+      command: 'createrepo .'
```

Comment:
Is there a difference between running `createrepo .` with chdir vs `createrepo {{ inventory_dir }}/downloaded_rpms`?

Reply:
I don't think there is.